src/helpers/resolve-app-function.ts

```diff
@@ -1,3 +1,7 @@
+import { createRequire } from "node:module";
+
+// Replace with import.meta.resolve when available in all supported Node.js versions
+const require = createRequire(import.meta.url);
```

Comment:
Unfortunately this isn't the best solution.

Currently `import.meta.resolve()` only accepts a second parameter (the basedir) with the `--experimental-import-meta-resolve` flag

`require.resolve()` won't return the ESM module of an app.

Using `import.meta.resolve` allows us to simplify by removing the need for the `appFnWithFileProtocol` as it returns a `file://` URL already

Reply:
What do you think about using [a Polyfill](https://github.com/wooorm/import-meta-resolve)?

Reply:
Sounds good to me.